app/forms/add_to_order_form.rb

```diff
@@ -66,6 +66,24 @@ def available_accounts
     AvailableAccountsFinder.new(original_order.user, current_facility)
   end
 
+  # We're using the ordered_at of the order details to determine if additional OrderDetails
+  # have been added to the order. We find the most recently added order that does not
+  # match the original order's ordered_at and use that for our basis for defaults.
+  # There is an edge case: 1) Upload an order via bulk upload. 2) Add on to that order
+  # here. 3) Upload again via bulk upload matching the ordered_at of the original.
+  # In that situation, we will return the results of #2.
+  def previously_added_order_detail
```

Comment:
I felt a little weird making this `public` just for testing purposes, but it made my confidence in the tests much stronger.